tensorflow/lite/micro/kernels/flex_add_v2.cc

```diff
@@ -28,7 +28,9 @@ limitations under the License.
 #include "tensorflow/lite/micro/memory_helpers.h"
 #include "tensorflow/lite/micro/micro_error_reporter.h"
 
-namespace tflite {
+namespace {
+
+using namespace tflite;
 
 void EvalAdd(TfLiteContext* context, TfLiteNode* node, TfLiteAddParams* params,
```

Comment:
I see the issue - I think renaming these methods to EvalFlexAddV2 and such would be a cleaner fix to avoid this naming collision, wdyt?